app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -77,6 +77,9 @@ def fetch_domain_servers(domain)
           next unless child.type_path.end_with?(hawk_escape_id('Domain WildFly Server'))
           @eaps << child
 
+          server_config = @ems.inventory_client.get_config_data_for_resource child.path
+          child.properties.merge! server_config['value'] unless server_config['value'].nil?
```

Comment:
is this still needed if this PR is merged? https://github.com/hawkular/hawkular-client-ruby/pull/138